### ISSUE TITLE
[EPIC] Source Google Ads to GA

### ISSUE THREAD

**Issue by lazebnyi** [airbytehq/airbyte#11758]
### According to [checklist](https://docs.google.com/spreadsheets/d/1m41LRV8lVqDHve5XDEP_vLJIbu0Li4jPLxJ0Adg8D8c/edit#gid=1820065035):

🚨Please check MLP features documented to see if any issues have had more than 5 users requested. If so, please tag Andy on next steps.🚨
- [MLP](https://docs.google.com/document/d/1zOu9rBTHLg3JX3oI770B-WNaEkD88saKqp7PjG1Cqr0) improvements:

No issues requested by more than 5 users found
- All outstanding bug reports blocking GA:
   - [x] https://github.com/airbytehq/alpha-beta-issues/issues/49
   - [x] ~https://github.com/airbytehq/airbyte/issues/10913~ april 28, @sherifnada: no longer in scope since Airbyte will handle docs / copy updates
   - [x] ~https://github.com/airbytehq/airbyte/issues/10911~ april 28, @sherifnada: no longer in scope since Airbyte will handle docs / copy updates
   - [x] ~https://github.com/airbytehq/airbyte/issues/10392~ april 28, @sherifnada: no longer in scope due to being blocked on the user. upd @davydov-d june 02: closed as duplicate of [12999](https://github.com/airbytehq/airbyte/issues/12999)
   - [x] https://github.com/airbytehq/airbyte/issues/12096
   - [x] https://github.com/airbytehq/airbyte/issues/12486
   - [x] https://github.com/airbytehq/airbyte/issues/12999

- Label connector as GA in cloud & oss:
   - [x] [label google ads as GA in the UI ](https://github.com/airbytehq/airbyte-cloud/pull/1800)
   - [x] https://github.com/airbytehq/airbyte/pull/13441

<details><summary><strong>Acceptance criteria </strong></summary>

- [x] Create a checklist document that should rbe placed on the [Google Drive](https://drive.google.com/drive/folders/1qOzblo8xiBJc8AiK4AyuDTC1Ilt-GFzu)
- [x] Checklist filled up by engineering team
- [x] Collect and list all tech debt-related issues in the section below
- [x] Create/Revise MLP document and place it [here](https://github.com/airbytehq/airbyte/issues/11287#issue-1175260119)
- [x] Move checklist to the unblockers team
- [x] Reviewed/Updated by unblockers team
- [x] Move checklist to the product team for review
- [x] Reviewed/Updated by product team
- [x] Update connector documentation according to the comments in the checklist
- [x] Certification completed
 
</details>

**Comment by sherifnada** [comment]
@davydov-d like [discussed in slack](https://airbytehq-team.slack.com/archives/C02U9R3AF37/p1651125026188399) message, I've crossed off the two documentation related issues from this epic. They will be handled by Airbyte

**Comment by misteryeo** [comment]
@davydov-d Checklist reviewed - LGTM after we close https://github.com/airbytehq/airbyte/issues/12486 and https://github.com/airbytehq/alpha-beta-issues/issues/49!

**Comment by davydov-d** [comment]
> @davydov-d Checklist reviewed - LGTM after we close #12486 and [airbytehq/alpha-beta-issues#49](https://github.com/airbytehq/alpha-beta-issues/issues/49)!

great, thanks

**Comment by lazebnyi** [comment]
Successfully certified to GA - https://docs.google.com/spreadsheets/d/1m41LRV8lVqDHve5XDEP_vLJIbu0Li4jPLxJ0Adg8D8c
